src/docs/platforms/python/chalice.mdx

```diff
@@ -0,0 +1,53 @@
+---
+draft: false
+categories: []
+toc: true
+title: Chalice
+sidebar_order: 2
+tags: []
+---
+The Chalice integration adds support for the [Chalice aws](https://aws.github.io/chalice/).
+The integration is avalible since 1.16.0
```

Comment:
Wrong version btw, let's do 0.17

Reply:
I mean the version 1.16.0 of chalice. What version 0.17 is related to?

Reply:
SDK version, adding minimum supported Chalice version is also good to have